common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2481,11 +2481,15 @@ StScrollBar {
   .modal-dialog-button-box { spacing: 3px; }
   .modal-dialog-button {
     padding: 3px 18px;
+    @include button(osd);
+    &:hover,&:focus { @include button(osd-hover); }
+    &:active { @include button(osd-active); }
+    &:insensitive { @include button(osd-insensitive); }
     &:default {
-      @include button(normal);
-      &:hover,&:focus { @include button(hover); }
-      &:active { @include button(active); }
-      &:insensitive { @include button(insensitive); }
+      @include button(osd-normal);
+      &:hover,&:focus { @include button(osd-hover); }
+      &:active { @include button(osd-active); }
+      &:insensitive { @include button(osd-insensitive); }
```

Comment:
This whole thing could be replaced with simply:
```
&, &:default {
  @extend %osd_button;
}
```

Reply:
Fair point :)